lib/src/abstract_span.dart

```diff
@@ -37,6 +37,9 @@ abstract class Span {
   /// Each span has a UTC timestamp describing when it started.
   DateTime get startTime;
 
+  /// Each span has a UTC timestamp describing when it started.
+  set startTime(DateTime value);
```

Comment:
There are times where we want to set the startTime.  For example, in the case of capturing the duration and sequence of events from a user navigating to the app until Home has opened requires using a browser api that tells us when our main span should have started, but after it has already been created.